src/sentry/seer/endpoints/organization_seer_explorer_runs.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
+
+import logging
+
+import orjson
+from django.conf import settings
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.models.organization import Organization
+from sentry.net.http import connection_from_url
+from sentry.seer.seer_setup import has_seer_access_with_detail
+from sentry.seer.signed_seer_api import make_signed_seer_api_request
+
+logger = logging.getLogger(__name__)
+
+autofix_connection_pool = connection_from_url(settings.SEER_AUTOFIX_URL)
+
+
+class OrganizationSeerExplorerRunsPermission(OrganizationPermission):
+    scope_map = {
+        "GET": ["org:read"],
+    }
+
+
+class ExplorerRunsRequestSerializer(serializers.Serializer):
+    limit = serializers.IntegerField(required=False, allow_null=False)
+
+
+@region_silo_endpoint
+class OrganizationSeerExplorerRunsEndpoint(OrganizationEndpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.ML_AI
+    enforce_rate_limit = True
+    permission_classes = (OrganizationSeerExplorerRunsPermission,)
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        """
+        Get the current state of a Seer Explorer session.
+        """
```

Comment:
don't we already have an endpoint for this? or is this comment inaccurate? doesn't seem to reflect the logic below